CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,3 @@
+Change package name
+FIX Long rule set
```

Comment:
In this case, given this PR is fixing an issue "with number" it could be included in the entry. I.e.:

"FIX Long rule set (#6)"